wg-advisors/why/why_branding_licencing.md

```diff
@@ -1,13 +1,68 @@
 # Why we should look after the project's brand and licencing
 
-TODO:
-
 ## Summary
 
+The Apache Software Foundation's most important asset is a combination of branding and license that is
+used by all Apache projects. The license is the Apache License, Version 2.0, and the branding is based
+on the project name. The licence is a permissive open source license that allows the software to be used
+by anyone for any purpose, and the branding is a set of rules that ensure that the software is recognized
+as an Apache project and that the project is recognized as part of the Apache Software Foundation.
+
+The software that ASF projects produce is released under the Apache License, Version 2.0, which is a permissive
+open source license that allows the software to be used by anyone for any purpose and it is important that
```

Comment:
This paragraph duplicates much of what is said in the first paragraph. Can we consolidate a little here?

Reply:
Yes. Merged the two. From the second paragraph I left only the "owning" part.